UnityProject/Assets/Scripts/Systems/Inventory/DynamicItemStorage.cs

```diff
@@ -583,6 +619,13 @@ public void Add(IDynamicItemSlotS bodyPartUISlots)
 			ServerObjectToSlots[bodyPartUISlots.GameObject].Add(Slot);
 
 			ServerTotal.Add(Slot);
+			var InternalData = new InternalData();
```

Comment:
if you make those properties instead of public fields you can do this
````cs
var internalData = new InternalData() 
{
    Id = id,
    IndexEnabled = i
}
```

Reply:
also notice I used camelCase for the privates and iterator of the loop

Reply:
ok cool will do